styles/stylesheet.css

```diff
@@ -133,58 +135,193 @@ h3 {
     padding: 20px;
 }
 
-.staff {
-
-    width: 60%;
-    float: left;
-    border: 2px solid #444444;
+.padding {
     margin-top: 20px;
+    text-align: center;
+}
+
+/* Lessons Page */
+.classes {
     display: grid;
```

Comment:
Good work on your grid layouts and styles. I usually try to avoid using columns/ rows for spacing and instead use row/column gaps and margins/flex reserving grid cells for elements and content only. But this only personal preference.